### PR TITLE
Vairous typo fixes/improving consistency

### PR DESCRIPTION
**Pull request by stuntguy3000**
Simply, this PR
- Improves consistency among certain classes
- Fixes grammatical/misleading information
- Correct branding of Minecraft

### CODE REVIEW

File: src/main/java/org/bukkit/PortalType.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package org.bukkit;
 
 /**
- * Represents various types of portals that can be made in a world.
+ * Represents various types of portals that can exist in a world.
```
Comment (Wolvereness) [inline src/main/java/org/bukkit/PortalType.java]: Where is PortalType returned as a value?

File: src/main/java/org/bukkit/Location.java [hunk 2]
```diff
@@ -377,9 +377,9 @@ public Location subtract(Vector vec) {
      * orientation independent.
      *
      * @see Vector
-     * @param x X coordinate
-     * @param y Y coordinate
-     * @param z Z coordinate
+     * @param x x coordinate
+     * @param y y coordinate
+     * @param z z coordinate
```
Comment (Wolvereness) [inline src/main/java/org/bukkit/Location.java]: Each of these should be `the _ coordinate`

File: src/main/java/org/bukkit/PortalType.java (hunk 1 above)
Comment (stuntguy3000) [inline src/main/java/org/bukkit/PortalType.java]: What do you mean?

I made the change because not all portals can be made without another plugin's interactions, which "can be made" kind of confusing.

Comment (Wolvereness) [inline src/main/java/org/bukkit/PortalType.java]: Unless a portal can _exist_ (as in, can be viewed), then it shouldn't indicate _exist_. The Javadoc implies that it is only used a parameter, so I'm requesting information to show the current Javadoc is wrong.

tl;dr: Portals don't exist in the API - they are only created.

File: src/main/java/org/bukkit/Location.java [hunk 3]
```diff
@@ -31,9 +31,9 @@ public Location(final World world, final double x, final double y, final double
      * Constructs a new Location with the given coordinates and direction
      *
      * @param world The world in which this location resides
-     * @param x The x-coordinate of this new location
-     * @param y The y-coordinate of this new location
-     * @param z The z-coordinate of this new location
+     * @param x The x coordinate of this new location
+     * @param y The y coordinate of this new location
+     * @param z The z coordinate of this new location
```
Comment (Wolvereness) [inline src/main/java/org/bukkit/Location.java]: These lines (and the diff just above) should have a lowercase <i>t</i>he.

**Comment by stuntguy3000** [comment]
Updated.

File: src/main/java/org/bukkit/Location.java [hunk 4]
```diff
@@ -31,11 +31,11 @@ public Location(final World world, final double x, final double y, final double
...
      * @param world The world in which this location resides
-     * @param x The x-coordinate of this new location
-     * @param y The y-coordinate of this new location
-     * @param z The z-coordinate of this new location
-     * @param yaw The absolute rotation on the x-plane, in degrees
-     * @param pitch The absolute rotation on the y-plane, in degrees
+     * @param x the x coordinate of this new location
+     * @param y the y coordinate of this new location
+     * @param z the z coordinate of this new location
+     * @param yaw the absolute rotation on the x-plane, in degrees
```
Comment (aerouk) [inline src/main/java/org/bukkit/Location.java:37]: Surely <code>x-plane</code> should follow the same format as <code>x coordinate</code>? @Wolvereness

Comment (stuntguy3000) [inline src/main/java/org/bukkit/Location.java:37]: It should, waiting for his response.
